source/common/router/router_ratelimit.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+#include "envoy/router/router.h"
+#include "envoy/router/router_ratelimit.h"
+
+#include "common/http/filter/ratelimit.h"
+
+namespace Router {
+
+/**
+* Action for service to service rate limiting.
+*/
+class ServiceToServiceAction : public RateLimitAction {
+public:
+  // Router::RateLimitAction
+  void populateDescriptors(const Router::RouteEntry& route,
```

Comment:
nit: all of the function declarations in this file should have full variable names.